index.html

```diff
@@ -271,7 +271,7 @@ <h3 id="key-concepts-and-terminology">Key Concepts and Terminology</h3>
 is an origin whose scheme component is <code>HTTPS</code>.</p>
 
     <p>The <dfn>message body</dfn> and the <dfn>transfer encoding</dfn> of a resource
-are defined by <a href="https://tools.ietf.org/html/rfc7230#section-3">RFC7230, section 3</a>. [[!RFC7230]] </p>
+are defined by <a href="https://tools.ietf.org/html/rfc7230#section-3">RFC7230, section 3</a>. [[!RFC7230]]</p>
 
     <p>The <dfn>representation data</dfn> and <dfn>content encoding</dfn> of a resource
 are defined by <a href="https://tools.ietf.org/html/rfc7231#section-3">RFC7231, section 3</a>. [[!RFC7231]]</p>
```

Comment:
It seems this is not needed as Fetch already handles the HTTP details and provides an abstraction on top of them. To then refer to HTTP details in this specification is somewhat weird.